libportal/portal-qt5.h

```diff
@@ -36,11 +36,17 @@ static inline gboolean _xdp_parent_export_qt (XdpParent *parent,
         return TRUE;
       }
     }
-
-  /* TODO: QtWayland doesn't support xdg-foreign protocol yet
-   * Upstream bugs: https://bugreports.qt.io/browse/QTBUG-73801
-   *                https://bugreports.qt.io/browse/QTBUG-76983
-   */
+  else
+    {
+      /* TODO: QtWayland doesn't support xdg-foreign protocol yet
+       * Upstream bugs: https://bugreports.qt.io/browse/QTBUG-73801
+       *                https://bugreports.qt.io/browse/QTBUG-76983
```

Comment:
Any movement on those?

Reply:
Unfortunately no and most likely this won't change in near future as the main QtWayland developer left Qt Company and QtWayland is now without a maintainer.

Reply:
This is a new feature, it's too late for Qt 5.15. Since it's the last 5.x release, you have to wait at least for Qt 6.0.
If somebody can make a patch there is people that will review and approve it even though Johan left.